news/_posts/translation/2021-11-09-do-you-speak-prestashop-october-2021.md

```diff
@@ -0,0 +1,121 @@
+
+![Norbert’s picture](/assets/images/2021/11/build-norbert-gift.jpg)
+
+If you have received your gift too, don’t forget to send me a picture at translation@prestashop.com so that I can share it in the next Crowdin report. 
+
+### 💕 Many thanks to newcomers
+
+Many people subscribe to the project(s) of their choice each month, but not so many start translating straight away, let alone become top translators! For that reason, we would like to thank the ones who rushed into the efforts immediately! So lots of love to the dedicated [Piotr A](https://crowdin.com/profile/piotr30/activity) and [slstranslation](https://crowdin.com/profile/slstranslation/activity).
+
+## A few stats
+ 
+* **18 members** were active on the project this month.
+* A total of **9,256 words have been translated** and **7,365 validated**.
+* All this in **15 different languages**.
+ 
+Thank you very much for your involvement!
+
+## Top contributors
+ 
+A lot of you are working every day on Crowdin to have PrestaShop available in many languages, and PrestaShop cannot thank you enough for your dedication! Here are the most active translators and proofreaders for October 2021.
+ 
+#### Top 10 translators in October:
+ 
+| |Translator | Language | # Words
+|-|---------- | -------- | ----------------
+| 1. |‫‬ 戢斌斌 (adonis_cn) | Chinese Simplified | 1,911
+| 2. | Zoran Tejic (zotamal) | Serbian (Latin) | 1,880
+| 3. | Piotr A (Piotr30) | Slovak | 1,269
+| 4. | Mahmoud Parvazi (m.parvazi) | Persian | 920
+| 5. | Girdman.com (anargaragezov) | Azerbaijani | 654
+| 6. | M. Mirena (mmirena) | Albanian | 467
+| 7. | slstranslation | Korean | 467
+| 8. | Rauno Riikman (weaver) | Finnish | 359
+| 9. | Rúben Martins (RubenMartins) | Portuguese | 334
+| 10. | Store (HelaBasa) | Sinhala | 287
+ 
+#### Top 8 proofreaders in October:
+ 
+| | Proofreader | Language | # Words
+|-| ---------- | -------- | ----------------
+| 1. | Zoran Tejic (zotamal) | Serbian (Latin) | 1,970
+| 2. | 戢斌斌 (adonis_cn) | Chinese Simplified | 1,911
+| 3. | webdvl (megashopba) | Slovak | 908
+| 4. | Store (HelaBasa) | Sinhala | 564
+| 5. | Girdman.com (anargaragezov) | Azerbaijaini | 559
+| 6. | M. Mirena (mmirena) | Albanian | 467
+| 7. | crowd-dyr | Danish | 310
+| 8. | Arman Hosseini (Arman-Hosseini) | Persian | 29
+
+
+Congratulations, and welcome to our new top contributors! :clap:
+ 
+Remember, you can see who has been contributing to our translation project thanks to the [Translators page](https://translators.prestashop.com/).
+ 
+## Complete translations
+ 
+#### Fully available languages
+ 
+At the end of October, PrestaShop 1.7.8 was already fully available (= 100% translated and validated) in **14 languages**. That’s great! Thank you for your involvement. :tada:
+ 
+| Danish | Chinese traditional | English | Estonian |
+| French | Greek | Italian | Lithuanian |
+| Portuguese, Brazil | Romanian | **Serbian** | Spanish |
+| Spanish, Mexico | Swedish |
```

Comment:
What's the layout you're looking for here? Because this won't work as a table as-is (it displays in a single line, see preview)

Reply:
It's supposed to be a table.  If you have a look at the previous Crowdin monthly, it always displays in a single line in the preview.

Reply:
Huh. Perhaps it's a jeykiill extension. As long as it works 🤷‍♂️